### ISSUE TITLE
Replace Files

### ISSUE THREAD

**Issue by PrecedingxD** [Minehut/Meta#778]
### Feature request

**Checklist**
- [ ] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ ] This is not a bug report, support request or plugin request
- [ ] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [ ] This issue does not contain any sensitive information 

**Platform**

**Describe the feature**

**How the feature is useful**

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

I haven't seen this already suggested, although I'm not too good at GitHub, so it may already be posted on here.

I'd like to request that replacing files is added. Currently, whenever you try to upload a file and it's already there, it says "File already exists.". Whenever a file is uploaded and it already exists, I personally think that there should be a popup with the following: "The file **plugin.jar** already exists!" and then buttons such as **Overwrite** and **Cancel**. 

Again, sorry if this was already suggested.

**Comment by PrecedingxD** [comment]
Whoops, didn't see the format posted. I'll edit it real quick.

**Comment by orvitpng** [comment]
Hell yeah, this is a great idea.

**Comment by emopedia** [comment]
yeah i would love this, especially when i transfer a lot of skripts over from vscode to the panel

**Comment by GStudiosX2** [comment]
Please

**Comment by aient** [comment]
Agreed, developers otherwise need to delete a file and upload it for every edit which can be a pain after a while.

**Comment by GStudiosX2** [comment]
they had overriding before in the old panel i have no idea why they removed it

**Comment by CoreyShupe** [comment]
The file manager was rewritten. This is planned.
